src/quoridor/server/server.go

```diff
@@ -48,11 +57,13 @@ func createGameHandler(w http.ResponseWriter, r *http.Request) {
 		response.SendBadRequestError(w, err)
 		return
 	}
-	game, err := gamecontroller.CreateGame(configuration)
+	authToken:= shortuuid.New()
+	game, err := gamecontroller.CreateGame(configuration, authToken)
 	if err != nil {
 		response.SendBadRequestError(w, err)
 		return
 	}
+	w.Header().Set(Authorization, authToken)
```

Comment:
You don't need a const here, or use "AuthorizationHeaderName" as const name. Because we can think that it's an error because we need a string here